FormFactory/AbstractMessageFormFactory.php

```diff
@@ -56,8 +56,6 @@ public function __construct(FormFactoryInterface $formFactory, AbstractType $for
      */
     protected function createModelInstance()
     {
-        $class = $this->messageClass;
-
-        return new $class();
+        return new $this->messageClass();
```

Comment:
I'm afraid it could fail if there is a method of the same name... But do as you want :)